packages/lambda-functions/project.json

```diff
@@ -4,17 +4,43 @@
   "sourceRoot": "packages/lambda-functions",
   "projectType": "application",
   "targets": {
-    "serve-functions": {
-      "command": "sam.cmd build && sam.cmd local start-api",
+    "build": {
+      "executor": "@nx/webpack:webpack",
+      "outputs": ["{options.outputPath}"],
       "options": {
-        "cwd": "packages/lambda-functions"
+        "compiler": "babel",
+        "target": "node",
+        "externalDependencies": "none",
+        "outputPath": "dist/packages/lambda-functions",
+        "main": "packages/lambda-functions/functions/hello-world/app.ts",
+        "tsConfig": "packages/lambda-functions/tsconfig.json",
+        "assets": [
+          {
+            "input": "packages/db/src/",
+            "glob": "schema.prisma",
+            "output": "/"
+          },
+          {
+            "input": "node_modules/.prisma/client",
+            "glob": "libquery_engine-rhel-openssl-*.so.node",
+            "output": "/"
+          }
+        ]
       }
```

Comment:
I'm using webpack to build the lambda. This gets us two important benefits:
* This target is dependent on the `db` package, so nx will build that first.
* We can include the prisma schema and the engine shared library as assets and webpack will include them in the build.